### ISSUE TITLE
Authenticates OK but then dies.

### ISSUE THREAD

**Issue by shokk** [mopidy/mopidy-gmusic#74]
gmusicapi.log:

```
logging to: /Users/xxxxx/Library/Logs/gmusicapi/gmusicapi.log
2015-03-22 20:14:23,510 - gmusicapi.Mobileclient1 (shared:70) [INFO]: initialized
2015-03-22 20:14:23,511 - gmusicapi.Mobileclient1 (shared:93) [INFO]: logged out
2015-03-22 20:14:23,563 - gmusicapi.Mobileclient1 (shared:204) [DEBUG]: ClientLogin(<omitted>)
2015-03-22 20:14:23,728 - gmusicapi.Mobileclient1 (shared:201) [DEBUG]: Init(args=[], kwargs={})
2015-03-22 20:14:24,500 - gmusicapi.Mobileclient1 (shared:242) [DEBUG]: 
2015-03-22 20:14:24,501 - gmusicapi.Mobileclient1 (mobileclient:43) [INFO]: authenticated
2015-03-22 20:14:24,503 - gmusicapi.Mobileclient1 (shared:201) [DEBUG]: ListTracks(args=[], kwargs={'start_token': None, 'max_results': 20000})
2015-03-22 20:14:33,654 - gmusicapi.Mobileclient1 (shared:242) [DEBUG]: {u'kind': u'sj#trackList', u'data': {u'items': ['<13621 tracks>']}}
```

logging with -vvv:

```
INFO     2015-03-22 20:14:24,501 [3426:GMusicBackend-4] gmusicapi.Mobileclient1
  authenticated
DEBUG    2015-03-22 20:14:24,503 [3426:GMusicBackend-4] gmusicapi.Mobileclient1
  ListTracks(args=[], kwargs={'start_token': None, 'max_results': 20000})
INFO     2015-03-22 20:14:24,505 [3426:GMusicBackend-4] requests.packages.urllib3.connectionpool
  Starting new HTTPS connection (1): www.googleapis.com
DEBUG    2015-03-22 20:14:30,312 [3426:GMusicBackend-4] requests.packages.urllib3.connectionpool
  "POST /sj/v1.5/trackfeed?alt=json&updated-min=0&include-tracks=true HTTP/1.1" 200 None
DEBUG    2015-03-22 20:14:33,654 [3426:GMusicBackend-4] gmusicapi.Mobileclient1
  {u'kind': u'sj#trackList', u'data': {u'items': ['<13621 tracks>']}}
ERROR    2015-03-22 20:14:33,846 [3426:GMusicBackend-4] pykka
  Unhandled exception in GMusicBackend (urn:uuid:fxxxx99e-6097-4625-9a15-xxxxxec61715):
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/pykka/actor.py", line 191, in _actor_loop
    self.on_start()
  File "/usr/local/lib/python2.7/site-packages/mopidy_gmusic/actor.py", line 30, in on_start
    self.library.refresh()
  File "/usr/local/lib/python2.7/site-packages/mopidy_gmusic/library.py", line 124, in refresh
    self._to_mopidy_track(song)
  File "/usr/local/lib/python2.7/site-packages/mopidy_gmusic/library.py", line 207, in _to_mopidy_track
    artists=[self._to_mopidy_artist(song)],
  File "/usr/local/lib/python2.7/site-packages/mopidy_gmusic/library.py", line 233, in _to_mopidy_artist
    name = song['artist']
KeyError: u'artist'
DEBUG    2015-03-22 20:14:33,847 [3426:GMusicBackend-4] pykka
  Unregistered GMusicBackend (urn:uuid:fxxxx99e-6097-4625-9a15-xxxxxec61715)
ERROR    2015-03-22 20:14:33,847 [3426:MainThread] mopidy.commands
  Uncaught exception
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/mopidy/commands.py", line 273, in run
    core = self.start_core(mixer, backends)
  File "/usr/local/lib/python2.7/site-packages/mopidy/commands.py", line 355, in start_core
    return Core.start(mixer=mixer, backends=backends).proxy()
  File "/usr/local/lib/python2.7/site-packages/pykka/actor.py", line 93, in start
    obj = cls(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/mopidy/core/actor.py", line 41, in __init__
    self.backends = Backends(backends)
  File "/usr/local/lib/python2.7/site-packages/mopidy/core/actor.py", line 112, in __init__
    has_library = b.has_library().get()
  File "/usr/local/lib/python2.7/site-packages/pykka/future.py", line 299, in get
    exec('raise exc_info[0], exc_info[1], exc_info[2]')
  File "<string>", line 1, in <module>
ActorDeadError: GMusicBackend (urn:uuid:fxxxx99e-6097-4625-9a15-xxxxxec61715) stopped before handling the message
INFO     2015-03-22 20:14:33,849 [3426:MainThread] mopidy.commands
  Stopping Mopidy frontends
DEBUG    2015-03-22 20:14:33,849 [3426:MainThread] mopidy.utils.process
  Stopping 0 instance(s) of MpdFrontend
DEBUG    2015-03-22 20:14:33,849 [3426:MainThread] mopidy.utils.process
  Stopping 0 instance(s) of HttpFrontend
DEBUG    2015-03-22 20:14:33,849 [3426:MainThread] mopidy.utils.process
  Stopping 0 instance(s) of NotifierFrontend
DEBUG    2015-03-22 20:14:33,849 [3426:MainThread] mopidy.utils.process
  Stopping 0 instance(s) of ScrobblerFrontend
```

**Comment by tatoosh** [comment]
same problem:
pykka: Unhandled exception in GMusicBackend

**Comment by caffinatedmonkey** [comment]
I recently started having the same issue after changing my app-specific password.

**Comment by jodal** [comment]
Fixed by #24 (and probably #27 and #64 if those had been merged first).
